addons/bestja_detailed_reports/models.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields, api, exceptions
+from dateutil import parser
+
+
+class Project(models.Model):
+    _inherit = 'bestja.project'
+
+    detailed_reports = fields.One2many('bestja.detailed_report', inverse_name='project')
+    use_detailed_reports = fields.Boolean(required=True, string="Raporty szacunkowe do zbiórki żywności", default=True)
```

Comment:
This checkbox is checked when you want to use detailed reports, it's checked by default so that you don't have to remember about checking it in the organizations on lower levels.